presto-spi/src/test/java/io/prestosql/spi/connector/TestConnectorViewDefinition.java

```diff
@@ -94,8 +95,8 @@ public void testRoundTrip()
                 Optional.of("test_catalog"),
                 Optional.of("test_schema"),
                 ImmutableList.of(
-                        new ViewColumn("abc", parseTypeSignature("bigint")),
-                        new ViewColumn("xyz", parseTypeSignature("array(varchar(32))"))),
+                        new ViewColumn("abc", BIGINT.getTypeId()),
+                        new ViewColumn("xyz", TypeId.of("array(varchar(32))"))), // TODO: this should be derived from the getTypeId() of the corresponding ArrayType instance
```

Comment:
Why not use `new ArrayType(createVarcharType(32)).getTypeId()`?

Reply:
Indeed. I only later realized I can just instantiate the type directly (which I can't do with Maps)